llvm/include/llvm/CodeGen/AccelTable.h

```diff
@@ -288,7 +310,19 @@ class DWARF5AccelTable : public AccelTable<DWARF5AccelTableData> {
   void convertDieToOffset() {
     for (auto &Entry : Entries) {
       for (AccelTableData *Value : Entry.second.Values) {
-        static_cast<DWARF5AccelTableData *>(Value)->normalizeDIEToOffset();
+        DWARF5AccelTableData *Data = static_cast<DWARF5AccelTableData *>(Value);
+        if (!Data->isNormalized())
```

Comment:
Maybe a comment here explaining that TU entries are normalized early, so some entries (TU entries) might already be normalized here - and we're normalizing the remaining CU entries, if that's the case.